applications/zpc/components/zpc_rust/src/zwave_poll_manager/zwave_poll_register.rs

```diff
@@ -38,8 +38,9 @@ const ATTRIBUTE_HOME_ID: AttributeTypeId = 0x2;
 const ATTRIBUTE_ENDPOINT_ID: AttributeTypeId = 0x4;
 
 //< This represents the Network Status of a node. NodeStateNetworkStatus
-const DOTDOT_ATTRIBUTE_ID_STATE_NETWORK_STATUS: AttributeTypeId = 0x000D;
-const ZCL_NODE_STATE_NETWORK_STATUS_ONLINE_FUNCTIONAL: u8 = 1;
+const DOTDOT_ATTRIBUTE_ID_STATE_NETWORK_STATUS: AttributeTypeId = 0xfd020001;
+// This is reported as a u32 in the attribute store
```

Comment:
what makes you think that there bug is not there ?

Reply:
I'm not sure I understand your question. I've added the community post for more context in the PR